boards/shields/adafruit_winc1500/Kconfig.defconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 Kim Bøndergaard <kim.bondergaard@prevas.dk>, Prevas A/S
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_ADAFRUIT_WINC1500
+
+config SPI
```

Comment:
For portability purpose, and similarly to boards, shields should provide a valid hw configuration, but not enable features (this is application responsibility).
So this configuration should be defined under `if NETWORKING`

Reply:
config SPI (and some other) moved inside if NETWORKING